evergreen/install_baas.sh

```diff
@@ -60,14 +60,14 @@ case $(uname -s) in
         case $DISTRO_NAME in
             ubuntu | linuxmint)
                 MONGODB_DOWNLOAD_URL="http://downloads.10gen.com/linux/mongodb-linux-$(uname -m)-enterprise-ubuntu${DISTRO_VERSION_MAJOR}04-5.0.3.tgz"
-                STITCH_ASSISTED_AGG_LIB_URL="https://stitch-artifacts.s3.amazonaws.com/stitch-mongo-libs/stitch_mongo_libs_ubuntu2004_x86_64_0bdbed3d42ea136e166b3aad8f6fd09f336b1668_22_03_29_14_36_02/libmongo-ubuntu2004-x86_64.so"
```

Comment:
I think that there's actually an ubuntu-specific build of these. https://evergreen.mongodb.com/task/stitch_mongo_libs_ubuntu2004_x86_64_assisted_agg_df9a68d900d7faddcee16bf0f532437da815a1c3_22_09_16_14_59_45